tests/tests_insights_client_register.yml

```diff
@@ -0,0 +1,41 @@
+# SPDX-License-Identifier: MIT
+---
+- name: Skip when running with embedded Candlepin
+  meta: end_play
+  when:
+    - lookup('env', 'LSR_RHC_TEST_DATA') | length == 0
```

Comment:
this needs to be moved as first task in `tasks`, as it does not work here:
```
ERROR! 'meta' is not a valid attribute for a Play

The error appears to be in '$PWD/rhc/tests/tests_insights_client_register.yml': line 3, column 3, but may
be elsewhere in the file depending on the exact syntax problem.

The offending line appears to be:

---
- name: Skip when running with embedded Candlepin
  ^ here
```
also, since the envvar is in the system where ansible runs (and not on the managed node), it needs also:
```yaml
      delegate_to: 127.0.0.1
      become: false
```